packages/smooth_app/lib/pages/scan/ml_kit_scan_page.dart

```diff
@@ -182,7 +182,9 @@ class MLKitScannerPageState extends State<_MLKitScannerPageContent> {
     }
   }
 
-  Future<void> _stopImageStream() async {
+  Future<void> _stopImageStream({
+    bool fromDispose = false,
```

Comment:
I don't mean to be rude, but as you don't use `fromDispose` later in the code, the gain is probably limited.

Reply:
OMG, the main statement for this PR is missing >_<
I'll fix this

Reply:
OMG, the main statement for this PR is missing >_<
I'll fix this